extension/popup.html

```diff
@@ -11,12 +11,12 @@
       href="https://fonts.googleapis.com/css2?family=Inter:wght@400;500;600&display=swap"
     />
   </head>
-  <body>
+  <body class="light-theme">
     <div class="container">
       <header class="header">
         <h1>LLMFeeder</h1>
-        <div class="active">
-          <div id="lightTheme">
+        <div class="theme-container">
+          <div class="dark-icon">
```

Comment:
Instead of using a div for theme toggler, it will be better to use a button because a button is more suited for clicking purpose. It will also improve semantics and accessbility.